addons/hr_payroll/tests/test_benefit.py

```diff
@@ -0,0 +1,337 @@
+# # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+import pytz
+from dateutil.relativedelta import relativedelta
+from odoo import exceptions
+from odoo.tests.common import tagged
+from odoo.addons.hr_payroll.tests.common import TestPayslipBase
+
+
+@tagged('benefit')
+class TestBenefit(TestPayslipBase):
+
+    def setUp(self):
+        super(TestBenefit, self).setUp()
+        self.tz = pytz.timezone(self.richard_emp.tz)
+        self.start = datetime.strptime('2015-11-01 01:00:00', '%Y-%m-%d %H:%M:%S').replace(tzinfo=self.tz)
```

Comment:
You should never need to use strptime(), the framework has all the bells and whistles to do that for you! E.g. `fields.Datetime.to_datetime()`. Please get rid of them all.